aztec/src/main/kotlin/org/wordpress/aztec/AztecText.kt

```diff
@@ -596,6 +601,44 @@ class AztecText : AppCompatEditText, TextWatcher, UnknownHtmlSpan.OnUnknownHtmlT
         return savedState
     }
 
+    private fun writeTempInstance(filename: String, obj: Any?) {
+        with(File.createTempFile(filename, ".inst", context.getCacheDir())) {
```

Comment:
Probably better catching `IOException` and `SecurityException` here.

Reply:
Good call @daniloercoli ! I've added guards with 8af62f6. (Note, it's on a different code line since the `createTempFile` code was removed with 68032e2.